SALT/Scripting/AnimCMD/Compiler.cs

```diff
@@ -135,7 +135,7 @@ public static ACMDCommand CompileSingleCommand(string line)
                     parameters[i] = param.Remove(0, param.IndexOf("=") + 1).Trim();
             }
 
-            var crc = ACMD_INFO.CMD_NAMES.Single(x => x.Value.Equals(name, StringComparison.InvariantCultureIgnoreCase)).Key;
+            var crc = ACMD_INFO.CMD_NAMES.Single(x => x.Value.Equals(name, StringComparison.InvariantCulture)).Key;
```

Comment:
Im not particularly sure its a good idea to make it case sensitive here. Being case-insensitive was a feature requested by many, but the primary reason i hadnt changed it back until now has been the inconsistant naming in the command dictionary. Some commands are CamalCase, others are camalCase, or just plain all lowercase.

That being said i think we can fix the inconsistant naming another time.